### ISSUE TITLE
Incorrect batch_norm usage

### ISSUE THREAD

**Issue by shatu** [geduo15/Very-Deep-Convolutional-Networks-for-Natural-Language-Processing-in-tensorflow#4]
1) The `trainable` argument in the batch_norm op should be replaced with `is_trainable`.
2) I think the starting convolution layer is incorrect. In my opinion, this stride should be [1, 1, ,1, 1] instead of [1, 1, embedding_size, 1], and `padding=Valid` instead of `padding=Same` should be used.
